src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -15,6 +15,9 @@ import { setup } from './util';
 
 import mockResponse from '../../src/mocks/response/idp/idx/introspect/default.json';
 
+const mockUsername = 'testuser@okta1.com';
+jest.mock('js-cookie', () => ({ get: () => mockUsername }));
```

Comment:
suggestion: we only mock these values on the tests where they are relevant

Reply:
I initially tried that but for some reason the mock did not take, it only triggered when I added it to the top of the test. I have seen this happen before with other tests so i thought it was a standard thing.

Reply:
Maybe I need to follow [these steps ](https://stackoverflow.com/questions/71470603/jest-mocks-only-work-at-top-of-file-never-inside-describe-or-it-blocks)to get the mock to work inside the `it`, I will give it a try.

Reply:
Ok so those steps did not work, and looking into their documentation it looks like jest works this way for ES module imports: https://jestjs.io/docs/manual-mocks#using-with-es-module-imports

However, it says using `jest.doMock` should work but trying it does not work. It seems like the automatic hoisting is causing this. I ended up using `spyOn` instead of mock to keep it scoped to the actual test using it.